client/src/components/Sidebar/Chat.js

```diff
@@ -46,9 +46,9 @@ class Chat extends Component {
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    setActiveChat: (id) => {
-      dispatch(setActiveChat(id));
-    },
+    setActiveChat: (username) => {
+      dispatch(setActiveChat(username));
+    }
```

Comment:
Good catch, but let's try and keep the changes specific to the current issue. If you want to make additional fixes or improvements, you can save those for your optional PR 🙂

Reply:
is it ok for me to leave it there for future PRs, or do you want me to revert it back to its original file ?